tests/manual-test-cases/Group18-VIC-UI/18-1-VIC-UI-Installer.md

```diff
@@ -1,32 +0,0 @@
-Test 18-1 - VIC UI Installation
```

Comment:
Do we need to recreate these tests in product build? If so, we need to add an issue to do so later.

Reply:
I think this is covered in https://github.com/vmware/vic-product/issues/1701.